pkg/metrics/metrics.go

```diff
@@ -0,0 +1,12 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net/http"
+)
+
+// ServeMetrics expose metrics
```

Comment:
How about writing something like this
```go
// ServeMetrics exposes metrics in Prometheus format
```

Reply:
Thanks! I have made the required changes.